src/tests/uat/platform_devices_uat.py

```diff
@@ -24,37 +22,72 @@
 """
 
 # Standard library imports
+import requests
 import sys
-from pprint import pprint
 
 # Internal library imports
 from cbc_sdk.helpers import build_cli_parser, get_cb_cloud_object
 from cbc_sdk.platform import Device
 
+HOSTNAME = ''
+ORG_KEY = ''
+HEADERS = {'X-Auth-Token': '', 'Content-Type': 'application/json'}
+SEARCH_DEVICES = '{}appservices/v6/orgs/{}/devices/_search'
+
+
+def search_devices_api_call():
```

Comment:
Instead of using global variables to pass around the data would it make more sense to pass the `cb` variable in as a parameter then move the following code inside the function
```
    HEADERS['X-Auth-Token'] = cb.credentials.token
    ORG_KEY = cb.credentials.org_key
    HOSTNAME = cb.credentials.url
```

Reply:
I am concerned that not using global variables would create code duplication in case we have several methods that perform API calls and need to invoke this information. Currently, this test has only one API call (respectively method) but thus we have it prepared to quickly add more.

Reply:
That's reasonable. Given this is a UAT test that is fine however we would not want to use global variables in the core sdk code. 

For reference, there are no planned changes to the device APIs at the moment and the other endpoints are device actions. Here is the API documentation if we want to add those to this UAT test https://developer.carbonblack.com/reference/carbon-black-cloud/platform/latest/devices-api/#device-actions

Reply:
I've added a Jira ticket to cover device actions.